helper-apps/cortex-file-handler/blobHandler.js

```diff
@@ -142,12 +142,14 @@ async function uploadBlob(
       });
 
       busboy.on("file", async (fieldname, file, info) => {
-        //set useGoogle if file is image or video
+        //do not use google if file is not image or video
         const ext = path.extname(info.filename).toLowerCase();
-        if (IMAGE_EXTENSIONS.includes(ext) || VIDEO_EXTENSIONS.includes(ext)) {
-          useGoogle = true;
+        const canUseGoogle = IMAGE_EXTENSIONS.includes(ext) || VIDEO_EXTENSIONS.includes(ext);
+        if(!canUseGoogle) {
+          useGoogle = false;
```

Comment:
Isn't this whole block just:

useGoogle = IMAGE_EXTENSIONS.includes(ext) || VIDEO_EXTENSIONS.includes(ext);

Reply:
there are two params "usegoogle" and "canusegoogle", it is just: if you received usegoogle for non-media case you do not need to consider it. so it is in two steps. yeah it can be done in single line like:

`useGoogle = (IMAGE_EXTENSIONS.includes(ext) || VIDEO_EXTENSIONS.includes(ext)) && useGoogle !== "false" && !!gcs;
`

but i think separating out as is current way is more human way.